### ISSUE TITLE
html is being escaped in event description fields

### ISSUE THREAD

**Issue by mfairchild365** [unl/UNL_UCBCN_Frontend#119]
See http://events-dev.unl.edu/s-mfairch4/2013/03/02/76596/
It appears that HTML was working in this field, and others (specifically the additional public details field).

**Comment by saltybeagle** [comment]
I don't believe many users knew of this possibility or took advantage of it. Supporting HTML output might be a nice feature, but it certainly opens up the app for XSS injection and other problems (from authenticated users of course).

I don't think it's a feature that MUST be kept. It may make more sense to auto-link URLs within plain text descriptions... I'm not sure.

**Comment by mfairchild365** [comment]
I agree, this needs some discussion.  I did a search on event descriptions and found ~250 (I don't have the exact number handy) events that have HTML hrefs.

**Comment by saltybeagle** [comment]
It would be interesting to know how many have urls within their descriptions, not wrapped in markup. I wonder if that's more widely used than embedding HTML markup.

Related issue on the old google code site: https://code.google.com/p/unl-event-publisher/issues/detail?id=2&can=1&q=html
